### PR TITLE
[MIG] base_user_role: Migration to 13.0

### PR DESCRIPTION
**Pull request by sebalix**

### CODE REVIEW

**Comment by jcdrubay** [comment]
Hi this is the gihub login of duc.dd@komit-consulting.com: https://github.com/novawish

The email address `ducdd@komit-consulting.com` is a typo.

File: base_user_role/data/ir_module_category.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ABF OSIELL <http://osiell.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record model="ir.module.category" id="ir_module_category_role">
+    <field name='name'>User roles</field>
```
Comment (jcdrubay) [inline base_user_role/data/ir_module_category.xml:14]: I don' think this is used anywhere, and I think that we should NOT create this module category as it seems like other similar modules in Odoo core are using the category Tools or Extra Tools.

File: base_user_role/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[ This file must be present and contains the usage instructions
+  for end-users. As all other rst files included in the README,
+  it MUST NOT contain reStructuredText sections
+  only body text (paragraphs, lists, tables, etc). Should you need
+  a more elaborate structure to explain the addon, please create a
+  Sphinx documentation (which may include this file as a "quick start"
+  section). ]
```
Comment (jcdrubay) [inline base_user_role/readme/USAGE.rst]: I guess you should remove this lines

**Review by jcdrubay — COMMENTED** [review]
Thank you @sebalix :) I have added 2 small comments. The rest LGTM.

**Comment by bealdav** [comment]
Hi @sebalix is it possible to add these commits in this PR ?

https://github.com/OCA/server-tools/pull/1489/commits

Not blocking for me but you know perfectly this module, it could be nice

Thanks in advance

File: base_user_role/data/ir_module_category.xml (hunk 1 above)
Comment (sebalix) [reply base_user_role/data/ir_module_category.xml:14]: This module category aims to be linked to role/group you create in your own custom module (https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/res_users.py#L101) if you want. By doing this you'll have the section 'User roles' in the "Access rights" tab on the user form which will regroup all groups related to role. Without it, all these groups will be in the 'Other' section.
I could add a comment above this XML record about that.

**Comment by bealdav** [comment]
Thanks @sebalix

**Review by jcdrubay — APPROVED** [review]

**Comment by sebalix** [comment]
@bealdav I'll continue to work on this the next week. There are a lot of fix and improvements in different branches, all of them need to be consolidated. It should be ok for 13.0 now (I just need to rebase some of them).
BTW I would like a tool/git command to compare commits between different branches based not on the commit hash (which has changed) but on other keys like author/date/message to know what to forward port/backport on a branch.

**Review by alan196 — APPROVED** [review]
Tested locally 👍 works as expected

**Comment by lmgr0312** [comment]
LGTM

**Review by lmgr0312 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
You have to rebase and pass black in pre-commit. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by eggu68** [comment]
@sebalix @pedrobaeza @Laurent-Corron what is the status of this migration? I see it did not pass all checks? Thank you for your good work.

**Comment by sebalix** [comment]
@pedrobaeza @eggu68 thank you to notice me about this. It's done, all checks are green now. Ready to be merged.

**Comment by eggu68** [comment]
@pedrobaeza can this be merged to 13.0 now? Thank you.

**Comment by pedrobaeza** [comment]
It seems there's a change in isort.cfg when merging. Maybe you need to rebase. There's an ongoing PR for improving this in https://github.com/OCA/maintainer-quality-tools/pull/629

**Comment by sebalix** [comment]
@pedrobaeza sorry for the delay, it's now rebased + isort.cfg updated to resolve the upcoming conflict. Ready to be merged.

**Pull request merged**